### PR TITLE
Enable serverless installation without istio DestinationRule CR

### PR DESCRIPTION
**Pull request by aerfio**
**Description**

Changes proposed in this pull request:

- as in title

**Related issue(s)**

### CODE REVIEW

File: resources/serverless/charts/docker-registry/values.yaml [hunk 1]
```diff
@@ -139,3 +139,6 @@ extraVolumes: []
 
 nameOverride:
 fullnameOverride:
+
+destinationRule:
+  enabled: true
```
Comment (dbadura) [inline resources/serverless/charts/docker-registry/values.yaml:144]: Add new line :)

Comment (aerfio) [inline resources/serverless/charts/docker-registry/values.yaml:144]:
```suggestion
  enabled: true
  
```

File: resources/serverless/charts/docker-registry/templates/destination-rule.yaml [hunk 2]
```diff
@@ -7,4 +8,5 @@ spec:
   host: {{ template "registry-fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
   trafficPolicy:
     tls:
-      mode: DISABLE
\ No newline at end of file
+      mode: DISABLE
+{{- end }}
```
Comment (aerfio) [inline resources/serverless/charts/docker-registry/templates/destination-rule.yaml:12]:
```suggestion
{{- end }}

```

File: resources/serverless/values.yaml [hunk 3]
```diff
@@ -185,6 +185,8 @@ dockerRegistry:
 
 docker-registry:
   fullnameOverride: "serverless-docker-registry"
+  destinationRule:
```
Comment (dbadura) [inline resources/serverless/values.yaml:188]: Do we need to add this value explicit?

Comment (aerfio) [reply resources/serverless/values.yaml:188]: good practices

File: resources/serverless/values.yaml [hunk 4]
```diff
@@ -132,9 +132,9 @@ containers:
       functionRequeueDuration:
         value: 5m
       functionBuildExecutorArgs:
-        value: "--insecure,--skip-tls-verify,--skip-unused-stages,--log-format=text,--cache=true"
+        value: "--insecure,--skip-tls-verify,--skip-unused-stages,--log-format=text,--cache=true,--verbosity=trace"
```
Comment (aerfio) [inline resources/serverless/values.yaml:135]:
```suggestion
        value: "--insecure,--skip-tls-verify,--skip-unused-stages,--log-format=text,--cache=true"
```

**Review by dbadura — APPROVED** [review]

**Pull request merged**